### ISSUE TITLE
Flatpak maintainer?

### ISSUE THREAD

**Issue by techguy16** [TravisEz13/PowerShell-flatpak#8]
Is there a maintainer for the Flatpak version of PowerShell? If not, I am willing to do it, but I will only do it for x.x releases, not minor releases.

**Comment by tazihad** [comment]
@techguy16 looks like abondoned. You should pick up and maintain it.

**Comment by techguy16** [comment]
Yeah probably I should. Errors happen too when building, debugging time 🤣
